modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceQuickAddToCart.testcase

```diff
@@ -1631,6 +1631,69 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-10475. User can paste text on quick add to cart search bar"
+	@priority = 3
+	test CanPasteTextOnQuickAddToCartSearchBar {
+		property portal.acceptance = "false";
+
+		task ("Given a business account with a user without any roles") {
+			CommerceEntry.addAccountEntryUser(
+				accountName = "Commerce Account",
+				accountType = "Business",
+				agreeToTermsAndAnswerReminderQuery = "true",
+				createNewAccount = "true",
+				requireReset = "false",
+				userEmailAddress = "testuser@liferay.com",
+				userFirstName = "Test",
+				userLastName = "User",
+				userScreenName = "testuser",
+				userSiteMembership = "Minium");
+		}
+
+		task ("And the user logs in") {
+			User.logoutPG();
+
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "testuser@liferay.com");
+		}
+
+		task ("When the user adds a product 'U-Joint' to cart and copies its SKU 'MIN55861'") {
+			Type(
+				locator1 = "Search#EMBEDDED_SEARCH_BAR",
+				value1 = "MIN55861");
+
+			Type.sendKeys(
+				locator1 = "Search#EMBEDDED_SEARCH_BAR",
+				value1 = "keys=CONTROL,a");
+
+			Type.sendKeys(
+				locator1 = "Search#EMBEDDED_SEARCH_BAR",
+				value1 = "keys=CONTROL,c");
+
+			Button.clickSubmitButton();
+
+			FrontStore.addInitializerProductToCart(
+				productName = "U-Joint",
+				productQuantity = 1);
+		}
+
+		task ("And pastes 'MIN55861' on the quick add-to-cart search bar") {
+			CommerceNavigator.gotoMiniCart();
+
+			Type.sendKeys(
+				locator1 = "Publications#KEYWORD_TEXTFIELD",
+				value1 = "keys=CONTROL,v");
+		}
+
+		task ("Then the pasted text is shown") {
+			AssertElementPresent(
+				locator1 = "Publications#KEYWORD_TEXTFIELD",
+				value1 = "MIN55861");
+		}
```

Comment:
@jaydawu if, following the paste, suggestions are shown, can you add an assertion for it? Thanks!